src/lib/raft/raft.h

```diff
@@ -151,6 +156,8 @@ struct raft {
 	uint32_t self;
 	/** Instance ID of leader of the current term. */
 	uint32_t leader;
+	/** The moment of the last communication with the leader. */
+	double leader_last_seen;
```

Comment:
Another entirely subjective comment from my side - I suggest to move this field to the other time-related fields in the end. For instance, right below `timer` field.

Reply:
I don't mind that. Fixed.